tests/extensions/test_base.py

```diff
@@ -0,0 +1,117 @@
+"""Tests extension base class methods"""
+from pathlib import Path
+from typing import Set, cast
+
+import pystac
+from pystac.extensions.base import ExtensionManagementMixin, PropertiesExtension
+from pystac.extensions.hooks import ExtensionHooks
+from pystac.extensions.projection import ProjectionExtension
+from pystac.extensions.raster import RasterExtension
+
+LOCAL_SCHEMA_URI = str(
+    Path("local", "path", "custom_extension", "v1.0.0", "schema.json")
+)
+
+
+class TestVersionHttpSchemaUri:
+    def test_get_version(self, item: pystac.Item) -> None:
+        item.stac_extensions = [
+            "https://stac-extensions.github.io/projection/v2.5.0/schema.json"
+        ]
+        assert ProjectionExtension.get_version(item) == "v2.5.0"
```

Comment:
```suggestion
        assert ProjectionExtension.get_version(item) == "2.5.0"
```

IMO the version should exclude the leading `v`, reference https://semver.org/#is-v123-a-semantic-version.